tests/test_e2e.py

```diff
@@ -277,6 +277,16 @@ def test_subject_name_issuer_authentication(self):
         self.assertIn('access_token', result)
         self.assertCacheWorksForApp(result, scope)
 
+    def test_pre_signed_jwt_authentication(self):
```

Comment:
```suggestion
    def test_client_assertion(self):
```

By the way, this suggestion can be accepted by one-click from the Github web interface.